module/VuFind/src/VuFind/Controller/AlphabrowseController.php

```diff
@@ -236,6 +238,11 @@ public function homeAction(): ViewModel
         $source = $this->params()->fromQuery('source', false);
         $from   = $this->params()->fromQuery('from', false);
         $page   = intval($this->params()->fromQuery('page', 0));
+        try {
+            $origin = new AlphaBrowseSearchOrigin($this->defaultTypes[$source], $source, $from, $page ?: null);
```

Comment:
You should use `$this->getTypes()` instead of `$this->defaultTypes` or else configuration will be ignored.